src/index.ts

```diff
@@ -1,28 +1,209 @@
-import puppeteer from "puppeteer";
-import { prompt } from "./utils/terminal";
+import { program } from "commander";
+import fs from "fs";
 import path from "path";
+import puppeteer from "puppeteer";
 import { loadData } from "./load-data";
+import { prompt } from "./utils/terminal";
+
+
+var fileValue;
+
 
-const jsonFileInput = path.resolve(process.cwd(), process.argv[2]);
+program
+	.arguments("<file>")
+	//this is here because for some reason it doesn't parse the file properly?
+	.action(function(file) {
+		fileValue = file;
+	})
+	
+	.option("-d, --debug",
+	"writes debug information to log file"
+	)
```

Comment:
The better option in my opinion is to chain all the `.option` first, and the `.action` will be the last, with all the rest of the code inside it, so you don't need to decalre a global `var fileValue`.